lib/widget/confirmation_input.dart

```diff
@@ -161,13 +207,30 @@ class ConfirmationInputState extends framework.WidgetState<ConfirmationInput> {
       ScreenController().executeAction(context, widget._controller.onComplete!);
     }
   }
+
+  @override
+  void clear() {
+    _otpPinFieldController.currentState?.clearOtp();
+  }
+
+  @override
+  void focusInputField() {
+    _otpPinFieldController.currentState?.hasFocus = true;
+    _otpPinFieldController.currentState?.focusNode.requestFocus();
+  }
+
+  @override
+  void unfocusInputField() {
+    _otpPinFieldController.currentState?.hasFocus = false;
+    _otpPinFieldController.currentState?.focusNode.unfocus();
+  }
 }
 
 extension FieldTypeOtpValue on String {
   OtpPinFieldDecoration get otpPinField {
     switch (this) {
```

Comment:
why not just have 2 options: underline & outline (default). To make it rounded just set the borderRadius.

Reply:
Currently, In that package, the borderRadius and borderWidth are used for some fieldType only, it's not for all the fieldTypes.

See all the fieldType's of Border - https://github.com/shivbo96/otp_pin_field/blob/4bff49ca5800b1f7a411ea7f8bd1d8b912bdf877/lib/src/otp_pin_field_state.dart#L245